index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+
+    <title>Maria's Blog</title>
+    <link rel="stylesheet" href="blogStyles.css">
+
+  </head>
+
+  <body>
+    <header>
+
+      <div id="blogTitle">
+        <h1>Maria's Blog</h1>
+        <h3>The shocking power of static pages</h3>
+      </div>
+
+      <nav>
+        <a href="sample/articles.com">Articles</a>
+        <a href="sample/about.com">About</a>
+        <a href="sample/twitter.com">Twitter</a>
+        <a href="https://github.com/MiaRomero">GitHub</a>
+      </nav>
+
+    </header>
+
+    <main>
+
+      <article>
+        <h2 class="articleTitle"></h2>
+        <p class="author"></p>
```

Comment:
Don't forget the author URL that's included in the blog article data so that the user has a link to more info on the author.